app/services/validators/validation-field-names.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  'hours': 'Contracted hours',
+  'reasonForReductionId': 'Reason for reduction',
+  'reduction-hours': 'Reduction hours',
+  'reduction_start_date': 'Reduction start date',
+  'reduction_end_date': 'Reduction end date',
```

Comment:
Field naming missing for `notes`. Error coming through as `undefined must be shorter than 4000 characters
 `